src/main/java/lotto/domain/LottoResults.java

```diff
@@ -1,30 +1,22 @@
 package lotto.domain;
 
 import java.math.BigDecimal;
+import java.math.RoundingMode;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 
 public class LottoResults {
 
     private static final int ZERO = 0;
```

Comment:
불필요한 import문 or 상수는 제거해주세요! 🙏
```suggestion
import java.util.Collections;
import java.util.List;
import java.util.Map;

public class LottoResults {

```

Reply:
헉 잔실수가 많네요 ㅠㅠ